content/includes/modal.html

```diff
@@ -50,22 +50,25 @@ <h5 class="va-modal-split-title">Servicemembers and Veterans applying for educat
 (function($){
   // TODO: eliminate jQuery dependency
   var modal = $('#va-modal').focus();
```

Comment:
Can you remove the `focus` call from this and add it inside the `open` function? I'm not sure why it was like this before, but it doesn't make sense to focus on the modal unless we've opened it.